suite/20-basic.sh

```diff
@@ -91,3 +91,49 @@ basic_learning_port()
     pass
 }
 alltests="$alltests basic_learning_port"
+
+basic_flags_flooding()
+{
+		require3loops
+		# Tests enabling and disabling of flooding unknown uni/multicast.
+		# Packet should always arrive on $h3. $h2 should only receive
+		# when flooding unknown is enabled on $b2.
+		create_br $br0 "" $bports
+
+		step "Verify unicast flood on"
+		bridge link set dev $b2 flood on
+		capture $h2 $h3
+		eth -i $h1 | { cat; echo from $h1; } | inject $h1
+		report $h2 | grep -q "from $h1" || fail
+		report $h3 | grep -q "from $h1" || fail
+
+		step "Verify unicast flood off"
+		bridge link set dev $b2 flood off
+		capture $h2 $h3
+		eth -i $h1 | { cat; echo from $h1; } | inject $h1
+		report $h2 | grep -q "from $h1" && fail
+		report $h3 | grep -q "from $h1" || fail
+
+		step "Verify multicast flood on"
+		# Craft multicast address
+		mcast_da="\001\000\136\252\252\252" # 01:00:5e:aa:aa:aa
+		mcast_sa="$(addrfmt $(rndaddr))"
+		et="$(be16 0xbbbb)"
+		mcast_target="${mcast_da}${mcast_sa}${et}"
+
+		bridge link set dev $b2 mcast_flood on
+		capture $h2 $h3
+		printf "$mcast_target" | { cat; echo from $h1; } | inject $h1
```

Comment:
So I think this could just be:

`eth -g 1 -i $h1 |  { cat; echo from $h1; } | inject $h1`